packages/turf-transform-translate/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@turf/transform-translate",
+  "version": "4.3.0",
+  "description": "turf transform-translate module",
+  "main": "index.js",
+  "types": "index.d.ts",
+  "files": [
+    "index.js",
+    "index.d.ts"
+  ],
+  "scripts": {
+    "test": "node test.js",
+    "bench": "node bench.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/Turfjs/turf.git"
+  },
+  "keywords": [
+    "turf",
+    "transform",
+    "transformation",
+    "translate",
+    "move",
+    "shift"
+  ],
+  "author": "Turf Authors",
+  "contributors": [
+    "Stefano Borghi <@stebogit>"
+  ],
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Turfjs/turf/issues"
+  },
+  "homepage": "https://github.com/Turfjs/turf",
+  "devDependencies": {
+    "@turf/rhumb-distance": "^4.3.0",
+    "@turf/distance": "^4.3.0",
+    "benchmark": "^2.1.4",
+    "load-json-file": "^2.0.0",
+    "tape": "^4.6.3",
+    "write-json-file": "^2.0.0"
+  },
+  "dependencies": {
+    "@turf/helpers": "^4.3.0",
+    "@turf/invariant": "^4.3.0",
+    "@turf/meta": "^4.3.0",
```

Comment:
Since `@turf/meta` does not exist in the `index.js` there's a new test in [`package/turf/test.js`](https://github.com/Turfjs/turf/blob/master/packages/turf/test.js#L109-L121) that will catch this as an error.